Lib/test/test_getpass.py

```diff
@@ -162,5 +162,65 @@ def test_falls_back_to_stdin(self):
             self.assertIn('Password:', stderr.getvalue())
 
 
+try:
+    import msvcrt
+    msvcrt_available = True
+except ImportError:
+    msvcrt_available = False
+
+@unittest.skipUnless(WIN32, "skipped on non-Windows platforms")
+@unittest.skipUnless(msvcrt_available, 'tests require system with msvcrt (Windows)')
```

Comment:
Let's use more consistent code
```suggestion
@unittest.skipUnless(msvcrt, 'tests require system with msvcrt (Windows)')
```